scripts/jenkins/ardana/ansible/roles/crowbar_setup/tasks/main.yml

```diff
@@ -2,7 +2,8 @@
 
 - name: Run the supplied qa_crowbarsetup.sh command on the admin node
   shell: |
-    . {{ admin_scripts_path }}/qa_crowbarsetup.sh
+    . {{ admin_mkcloud_config_file }}
```

Comment:
I would personally do it as a separate task around here: https://github.com/SUSE-Cloud/automation/blob/6d703163a543ba8f378eea05f64ce2641ef87e19/scripts/jenkins/ardana/ansible/install-crowbar.yml#L38 with explicit `failed_when`. wdyt?